.travis.yml

```diff
@@ -1,7 +1,7 @@
 # It may be tempting to add parens around each individual clause in this expression, but Travis then builds pushes anyway
-if: branch = master OR branch =~ ^features/ OR branch =~ ^feature- OR branch =~ ^release/ OR tag IS present
+if: branch = master OR branch =~ ^features/ OR branch =~ ^feature- OR branch =~ ^release/ OR tag =~ ^v\d+.*
 language: go
-go: 1.13.4
+go: 1.13.x
```

Comment:
@stack72 the upstream TF provider is now built & released with Go 1.14

Reply:
we are not using go 1.14 internally yet :) We will upgrade these soon!